README.md

```diff
@@ -10,169 +10,103 @@ _COSCRAD_ is a loose collaboration of several organizations, technical teams, an
-
-We provide an optional shell script for spinning up an ArangoDB instance. This will only work if you have docker installed on your development machine. Alternatively, you could run an ArangoDB instance on a VirtualBox VM, a local physical machine, or in the cloud.
+Our apps are of two kinds:
 
-#### Arango Setup Script
+**core coscrad applications**
 
-To fire up a disposable development instance of `ArangoDB` with docker, first from the config directory in the API project, copy sample.env to test.env and development.env in the same directory. Then run
+-   [`api`](./apps/api/README.md) (core back-end for the COSCRAD `web of knowledge`)
+-   [`coscrad-frontend`](./apps/coscrad-frontend/README.md) (standard front-end for the COSCRAD `web of knowledge`)
 
-> > npm run start:db:dev
+**community projects**
 
-from the project root.
+-   `tng-dictionary` (maintained by Justin Bambrick, TNG)
+-   `tsilqotin-language-hub` (maintained by Justin Bambrick, TNG)
+-   `tng-radio-hub` (maintained by Blake Sellars, TNG)
 
-The script will
+The core apps are developed and maintained by the core COSCRAD team. These projects
+are subject to strict guidelines for code quality and automated test coverage.
 
--   run a test that validates that our dummy data satisfy all domain invariants and
-    exports this data to json
--   spin up the Docker instance
--   bind the default arango port 8529 of the container to `$ARANGO_DB_PORT` on the host
--   add a database named `ARANGO_DB_NAME`
--   add all required collections to the database
--   add the user `ARANGO_DB_USER` with password `ARANGO_DB_USER_PASSWORD`
--   grant the user permissions on the database
+The community projects currently consist of several prototypes of front-end clients
+that leverage the COSCRAD API. The Many ideas explored in the community projects
+will eventually find their way into Coscrad's core.
 
-You will be asked to confirm the location of a persistent volume to be shared
-between the docker container and its host.
+**COSCRAD CLI**
+We also have built our own command-line interface, `COSCRAD CLI`. This is used
+for administrative tasks like dumping data snapshots (in persistence layer, domain layer
+and view layer formats), seeding test data and restoring data dumps, dumping
+a JSON version of schemas for our domain models, view models, command and event payloads,
+and so on.
 
-You will also be asked if you would like to seed the database. If you respond 'y',
-the database will be seeded with dummy data that we use for test. This dummy data
-respresents a comprehensive application state, including a collection of dummy
-data for every `entity type`. Further, all this data has passed though our
-domain invariant validation layer.
-
-To verify, your instance is running, open
+Currently, `COSCRAD CLI` is part of the `api`, and it is available via a custom
+build of the `api`. Run `npx nx build:api:cli` to build the `COSCRAD CLI`. The
```

Comment:
Move `npx nx...` command to its own line?

Reply:
In the future, commands could be moved out of the prose description and into a table with command and purpose columns.  Or some other format but basically more of a command reference.